### PR TITLE
Add new kirbytag-kbd plugin

### PR DESCRIPTION
**Pull request by scottboms**

### CODE REVIEW

File: content/plugins/scottboms/kirbytag-svg/plugin.txt [hunk 1]
```diff
@@ -26,6 +26,8 @@ Example:
 
 ----
 
+<<<<<<< HEAD
```
Comment (distantnative) [inline content/plugins/scottboms/kirbytag-svg/plugin.txt]: Looks you still have leftovers of a merge conflict in here

**Review by distantnative — CHANGES_REQUESTED** [review]

File: content/plugins/scottboms/kirbytag-svg/plugin.txt (hunk 1 above)
Comment (scottboms) [reply content/plugins/scottboms/kirbytag-svg/plugin.txt]: Oh, will have a look. Sorry about that, thought I fixed it.

**Comment by texnixe** [comment]
@scottboms This has still some conflict that prevents rebasing?

**Comment by scottboms** [comment]
> @scottboms This has still some conflict that prevents rebasing?

Does it? I'm not seeing any other conflicts here.

**Comment by scottboms** [comment]
Think I found the problems and should all be fixed now.

**Comment by bastianallgeier** [comment]
I think we are making your life harder here than it should be. Sorry for that. We are still working on the idea to have a proper plugin submit form. I just added your plugin manually.

**Pull request closed without merging**